wpilibNewCommands/src/main/native/include/frc2/command/SwerveFollowerCommand.inc

```diff
@@ -0,0 +1,94 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+#pragma once
+
+template <typename T>
+int sgn(T val) {
```

Comment:
You can use wpi::sgn() from wpi/MathExtras.h instead. Although, this doesn't even appear to be used so you can remove it.